WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -143,7 +146,8 @@
           <Border Classes="warning info">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction creates change." />
+              <TextBlock Text="Transaction creates change." 
+                         ToolTip.Tip="Creating change in a transaction is a privacy risk because it can link a following transaction that will use it with this one that created it. Use the change avoidance suggestion if possible, or coinjoin the change later." />
```

Comment:
Notice that according to the current implementation creating change is considered as a `warning info` and not a transaction privacy risk. For example if you use private coins to make a tx that creates change, the warnings icon would be an empty green shield.

```suggestion
                         ToolTip.Tip="Change can be used to link this transaction with the next one that will use it. If possible use the change avoidance suggestion, or coinjoin the change later." />
```